From 856159b3913976c68c8a04b7e939a6cc64477f6a Mon Sep 17 00:00:00 2001
From: Julien Nabet <serval2412@yahoo.fr>
Date: Thu, 22 Feb 2018 21:33:20 +0100
Subject: tdf#115828: hsqldb supports multiple resultSets

Regression from 3c567e55e6aed9e0da892d8b9fc09f8a0c784fcc
Thank you to Lionel for this suggestion

Change-Id: I5eeadc1968e57d586e55a7aa5488b2ab3ee43724
Reviewed-on: https://gerrit.libreoffice.org/50202
Reviewed-by: Lionel Elie Mamane <lionel@mamane.lu>
Tested-by: Jenkins <ci@libreoffice.org>

--- a/src/org/hsqldb/jdbc/jdbcDatabaseMetaData.java
+++ b/src/org/hsqldb/jdbc/jdbcDatabaseMetaData.java
@@ -1347,7 +1347,7 @@ public class jdbcDatabaseMetaData implem
      * @exception SQLException if a database access error occurs
      */
     public boolean supportsMultipleResultSets() throws SQLException {
-        return false;
+        return true;
     }
 
     /**
